Xplat/src/main/java/vazkii/botania/common/internal_caps/ItemFlagsComponent.java

```diff
@@ -29,14 +29,20 @@ public class ItemFlagsComponent extends SerializableComponent {
 	 * Used so certain mechanics don't interact with items immediately after they're produced.
 	 */
 	private int manaInfusionCooldown = 0;
+	/**
+	 * Similar to {@link #manaInfusionCooldown} but for the Runic Altar.
+	 */
+	private int runicAltarCooldown = 0;
 	public static final int INITIAL_MANA_INFUSION_COOLDOWN = 25;
+	public static final int INITIAL_RUNIC_ALTAR_COOLDOWN = 15;
```

Comment:
any rationale behind this number? if not, let's align it with the mana infusion number for consistency.

Reply:
Basically, I felt like it's slightly more likely someone would want to intentionally reuse outputs as inputs for the next craft compared to the mana pool. For example, a machine that produces one rune type. So I chose a lower number to not slow down such automation as much, while still giving the desired faster hopperhock pickup. 

...although for that to make any difference we would need to also lower the existing altar cooldown from 60 ticks.

Reply:
Ended up just making it the same.